app/code/Magento/Sales/Controller/Adminhtml/Order/AddressSave.php

```diff
@@ -51,4 +125,19 @@ public function execute()
             return $resultRedirect->setPath('sales/*/');
         }
     }
+    
+    /**
+     * Update region data
+     *
+     * @param array $attributeValues
+     * @return void
+     */
+    protected function updateRegionData(&$attributeValues)
```

Comment:
Passing arguments by reference is not recommended. Return modified data instead (possible memory usage optimization is not so important here).